xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml

```diff
@@ -1332,17 +1330,15 @@ The `password` will be converted to bytes using the UTF8 encoding. For
         <param name="salt">The key salt used to derive the key.</param>
         <param name="iterations">The number of iterations for the operation.</param>
         <param name="hashAlgorithm">The hash algorithm to use to derive the key.</param>
-        <param name="outputLength">The size of key to derive.</param>
+        <param name="outputLength">The size of key to derive (that is, the number of elements in the byte array).</param>
         <summary>Creates a PBKDF2 derived key from a password.</summary>
-        <returns>A byte array of length <paramref name="outputLength" /> that is filled with pseudo-random key bytes.</returns>
+        <returns>A byte array of length <paramref name="outputLength" /> that's filled with pseudo-random key bytes.</returns>
```

Comment:
Introducing a contraction? That surprises me, and feels off-voice.

Reply:
Microsoft style is for writing to be "warm and relaxed" and, at least in non-reference content, [contractions are preferred](https://learn.microsoft.com/en-us/product-style-guide-msft-internal/grammar-usage/contractions). But I'm unsure whether the guidance applies to reference docs, so I'll undo this.